cmd/organization.go

```diff
@@ -0,0 +1,238 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"github.com/spf13/cobra"
+	"github.com/uselagoon/lagoon-cli/pkg/output"
+	l "github.com/uselagoon/machinery/api/lagoon"
+	lclient "github.com/uselagoon/machinery/api/lagoon/client"
+	s "github.com/uselagoon/machinery/api/schema"
+)
+
+var addOrgCmd = &cobra.Command{
+	Use:     "organization",
+	Aliases: []string{"o"},
+	Short:   "Add a new organization to Lagoon",
+	PreRunE: func(_ *cobra.Command, _ []string) error {
+		return validateTokenE(lagoonCLIConfig.Current)
+	},
+	RunE: func(cmd *cobra.Command, args []string) error {
+		debug, err := cmd.Flags().GetBool("debug")
+		if err != nil {
+			return err
+		}
+		organizationName, err := cmd.Flags().GetString("organization")
+		requiredInputCheck("Organization name", organizationName)
+		if err != nil {
+			return err
+		}
+		organizationFriendlyName, err := cmd.Flags().GetString("friendlyName")
+		if err != nil {
+			return err
+		}
+		organizationDescription, err := cmd.Flags().GetString("description")
+		if err != nil {
+			return err
+		}
+		organizationQuotaProject, err := cmd.Flags().GetUint("quotaProject")
+		if err != nil {
+			return err
+		}
+		organizationQuotaGroup, err := cmd.Flags().GetUint("quotaGroup")
+		if err != nil {
+			return err
+		}
+		organizationQuotaNotification, err := cmd.Flags().GetUint("quotaNotification")
+		if err != nil {
+			return err
+		}
+		organizationQuotaEnvironment, err := cmd.Flags().GetUint("quotaEnvironment")
+		if err != nil {
+			return err
+		}
+		organizationQuotaRoute, err := cmd.Flags().GetUint("quotaRoute")
+		if err != nil {
+			return err
+		}
```

Comment:
maybe the flags for these would be better as `project-quota`, `environment-quota`, etc..
I know in the API they start with quota, but for flags it sorta makes more sense to swap the order